pkg/networkservice/core/trace/client.go

```diff
@@ -48,62 +46,50 @@ func NewNetworkServiceClient(traced networkservice.NetworkServiceClient) network
 
 func (t *beginTraceClient) Request(ctx context.Context, request *networkservice.NetworkServiceRequest, opts ...grpc.CallOption) (*networkservice.Connection, error) {
 	// Create a new logger
+	ctx = addIDCtx(ctx, request.GetConnection().GetId())
```

Comment:
Example of https://github.com/networkservicemesh/sdk/pull/1096#discussion_r724946768
```suggestion
	log.Fields(ctx)["id"] = id
```

Reply:
@Mixaster995